### PR TITLE
Switch out Kelly for Beth on the newsletter team

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

**Comment by bethaitman** [comment]
Closed because I got confused, ignore that 🙈

File: docs/team.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ Responsibilities
 Members
 ```````
 
-* `Kelly O'Brien <https://twitter.com/OBrienEditorial>`_ *
+* `Beth Aitman <https://twitter.com/baitman>`_ *
```
Comment (ericholscher) [inline docs/team.rst:122]: Maybe keep Kelly on the team for now, but add yourself with the `*`? Or maybe add a list of "past contributors" or some other way of acknowledging the work folks have done?

Comment (ericholscher) [reply docs/team.rst:122]:
```suggestion
* `Beth Aitman <https://twitter.com/baitman>`_ *
* `Kelly O'Brien <https://twitter.com/OBrienEditorial>`_ 
```

Comment (bethaitman) [reply docs/team.rst:122]: Good shout, have done. I do love this new suggestions feature!

**Review by mjang — APPROVED** [review]

**Pull request merged**